tests/foreman/ui/test_sync_templates.py

```diff
@@ -0,0 +1,125 @@
+"""Test class for Foreman Templates Import Export UI
+
+:Requirement: TemplatesPlugin
+
+:CaseAutomation: Automated
+
+:CaseLevel: Integration
+
+:CaseComponent: TemplatesPlugin
+
+:TestType: Functional
+
+:Upstream: No
+"""
+import pytest
+from fauxfactory import gen_string
+from nailgun import entities
+
+from robottelo import ssh
+from robottelo.decorators import tier2
+from robottelo.decorators import upgrade
+
+
+@pytest.fixture(scope='module')
+def templates_org():
```

Comment:
@jyejare can we use fixture `module_org` and `module_location` which is already available?

Reply:
Problem is UI tests has its own conftest, and `module_org` from that conftest return the `Default Organization` hence I would never be able to use the `module_org` from robottelo/api/entity_fixtures.py. So using local templates_org.

Since I am using templates_org, I created templates_loc to keep it simple, so people wont end up amazing why org local and loc from entity_fixtures. :)

I hope that makes sense.

Reply:
yes, it does. Thanks for clarifying.